keyboards/snampad/config.h

```diff
@@ -0,0 +1,245 @@
+/*
+Copyright 2019 REPLACE_WITH_YOUR_NAME
```

Comment:
You should put your name here, for all of the Snampad files.

```suggestion
Copyright 2019 Peter Tillemans
```